fineract-provider/src/main/resources/application.properties

```diff
@@ -164,3 +164,10 @@ fineract.activemq.standingInstructionInsufficientBalanceFailureQueue=${INSUFFICI
 fineract.configuration.resetNuban=${CONFIGURATION_RESET_NUBAN:false}
 fineract.configuration.nubanCode=${CONFIGURATION_NUBAN_CODE:00000}
 fineract.configuration.pentahoFolderName=${PENTAHO_FOLDER_NAME:pentahoReports}
+
+//odoo integration
+fineract.integrations.odoo.db=${ODOO_DB:odooprod}
+fineract.integrations.odoo.username=${ODOO_USERNAME:john@fiter.io}
+fineract.integrations.odoo.password=${ODOO_PASSWORD:pass}
+fineract.integrations.odoo.url=${ODOO_URL:http://41.242.142.227:6078}
```

Comment:
Should this be here? This looks like sensitive data. Why not put localhost or something? The actual details will be provided in the environment

Reply:
This is test derails but let me remove I was using then I left it here

Reply:
Fixed it please check now